# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/src/bindings.rs
# Commit: 32951c9cc186
# Full Hash: 32951c9cc186f044c7a59e0b132339139dfb9713
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2019-09-18 09:56:40
# Regressor Bug: 1570081
# File Overlap Count: 1
# Description:
#   Bug 1570081 - Support changing the blob image visible rect in WebRender. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D39911
# ==============================================================================

diff -r 37598891cea6 -r 32951c9cc186 gfx/webrender_bindings/src/bindings.rs
--- a/gfx/webrender_bindings/src/bindings.rs	Tue Sep 17 09:20:04 2019 +0000
+++ b/gfx/webrender_bindings/src/bindings.rs	Tue Sep 17 09:12:26 2019 +0000
@@ -3258,9 +3258,8 @@
 extern "C" {
      // TODO: figure out the API for tiled blob images.
      pub fn wr_moz2d_render_cb(blob: ByteSlice,
-                               width: i32,
-                               height: i32,
                                format: ImageFormat,
+                               render_rect: &LayoutIntRect,
                                visible_rect: &DeviceIntRect,
                                tile_size: Option<&u16>,
                                tile_offset: Option<&TileOffset>,